routers/repo/issue.go

```diff
@@ -925,6 +936,23 @@ func NewComment(ctx *context.Context, form auth.CreateCommentForm) {
 			(form.Status == "reopen" || form.Status == "close") &&
 			!(issue.IsPull && issue.PullRequest.HasMerged) {
 
+			// Check for open dependencies
+			noDeps, err := models.IssueNoDependenciesLeft(issue)
+			if err != nil {
```

Comment:
Please minimize that as shown above

Reply:
Doesn't work, i need access to `noDeps`, if I declare it before it wouldn't set the new value returend by `IssueNoDependenciesLeft()`...